packages/react-dom/src/__tests__/ReactErrorBoundaries-test.internal.js

```diff
@@ -784,7 +788,7 @@ describe('ReactErrorBoundaries', () => {
     ]);
   });
 
-  it('propagates errors on retry on mounting', () => {
+  xit('propagates errors on retry on mounting', () => {
```

Comment:
This is related to item 2 in the "Unresolved items" section. Waiting for feedback on whether this is too drastic a change and if we should hew more closely to the existing retry semantics.